### ISSUE TITLE
Move doc pages for how to read MatNWB files to MatNWB documentation

### ISSUE THREAD

**Issue by ehennestad** [NeurodataWithoutBorders/nwb-overview#180]
I am setting up a new documentation page for MatNWB here:

https://matnwb.readthedocs.io/en/latest/index.html

Perhaps it would be good to move the pages under `docs/source/file_read/matnwb` to to the new MatNWB docs?

**Comment by rly** [comment]
Yes, I think that's a great idea.

**Comment by oruebel** [comment]
Just to clarify, does this mean the GitHub page at https://neurodatawithoutborders.github.io/matnwb/ will be retired?

**Comment by ehennestad** [comment]
> Just to clarify, does this mean the GitHub page at https://neurodatawithoutborders.github.io/matnwb/ will be retired?

Yes. Should we keep it alive though and point to the new docs? There are many links pointing to it from other sites
